common/app/views/support/cleaner/AmpEmbedCleaner.scala

```diff
@@ -305,6 +305,22 @@ case class AmpEmbedCleaner(article: Article) extends HtmlCleaner {
       })
   }
 
+  def cleanAmpImg(document: Document): Unit = {
+    document
+      .getElementsByTag("img")
+      .tagName("amp-img") // rename <img> into <amp-img>
+      .foreach { img =>
+        img.attr("layout", "responsive")
+        // Setting img size
+        Seq("width", "height").foreach { attr =>
+          val value = Option(img.attr(attr))
+            .filter(_.nonEmpty)
+            .getOrElse("1") // Since layout is set to responsive any non empty/zero size would result in the image width being 100% of its container's width (keeping the same ratio)
```

Comment:
So this means if there is no height information then the ratio will be 1 : 1  for height and width?

Reply:
it means the `width` and `height` will be `1` which doesn't seem to matter when `layout=responsive` is set (as long is it not 0 or empty)

Reply:
But what it does mean is that there if the image is actually the ratio 5:3, then there will be a lot of extra space reserved by AMP which doesn't need to be there. And if the ratio is something like 1:2 then part of the image will be cut off.

That said, I think this is fine right now as I can't think of anything better to do really. Is there any standards to the witness image dimensions at all?

Reply:
Correction: @TBonnin and I just tested and actually it stretches the image to fit the ratio provided - it doesn't just reserve space/cut off  the image